spec/chartmogul/subscription_event_spec.rb

```diff
@@ -0,0 +1,196 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe ChartMogul::SubscriptionEvent do
+  describe 'API interactions', vcr: true do
+    it 'lists all subscription events', uses_api: true do
+      data_source = ChartMogul::DataSource.new(
```

Comment:
[duplication] This whole test setup is repeated and should go into a `before` block, please.

Reply:
Will do in a separate PR